### DISCUSSION TITLE
Remove beatmap button missing in multiplayer

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17534] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/40531871/160669878-4f7beac9-3d1d-4e5a-b9b0-1f4a5e47f726.png)

Sometimes the button to remove the map is missing, someone told me about it had to check for myself.
Not sure why

**Comment by AuracleTech** [comment]
From what I can test if you start a map then cancel it I believe the game thinks the map should be removed, but it doesn't get removed, so only the button disappear.

**Comment by bdach** [comment, marked as answer]
i believe this was intentional for the time being. the current playlist item cannot be removed yet.

**Reply by AuracleTech** [reply]
There's a remove button tho to remove them
The second one on the list has one, its just the first map that played when I cancelled that doesn't have it

**Reply by Theighlin** [reply]
The first map has no remove button, ever. That's how it is currently done. You will be able to remove it in the future.

**Reply by AuracleTech** [reply]
I seeeeeeee
